### PR TITLE
support resize image when reading image

### PR DESCRIPTION
**Pull request by dding3**
Fix intel-analytics/zoo#257

### CODE REVIEW

**Comment by jenniew** [comment]
Verified. It works.

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/nnframes/NNImageReaderSpec.scala [hunk 1]
```diff
@@ -124,4 +124,9 @@ class NNImageReaderSpec extends FlatSpec with Matchers with BeforeAndAfter {
     assert(imageOrigin == extractedOrigin)
   }
 
+  "image reader with resize" should "work" in {
+    val imageDF = NNImageReader.readImages(pascalResource.getFile, sc,
+      resizeH = 128, resizeW = 128)
+    assert(imageDF.count() == 1)
+  }
```
Comment (jenniew) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/nnframes/NNImageReaderSpec.scala]: Can you add assertion to check the height and width after resizing are correct?

Comment (jenniew) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/nnframes/NNImageReaderSpec.scala]: Can you also add test case for reading imageset with resize?

Comment (dding3) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/nnframes/NNImageReaderSpec.scala]: Added

**Comment by dding3** [comment]
Jekins passed http://10.239.47.211:18888/job/ZOO-PR-Validation/646/

Thanks for your review.

**Comment by jason-dai** [comment]
Need to update the doc

**Pull request merged**